composer.json

```diff
@@ -1,46 +1,52 @@
 {
-  "name": "laravel-doctrine/migrations",
-  "type": "library",
-  "description": "Doctrine Migrations for Laravel",
-  "license": "MIT",
-  "keywords": [
-    "doctrine",
-    "laravel",
-    "orm",
-    "data mapper",
-    "database",
-    "migrations"
-  ],
-  "authors": [
-    {
-      "name": "Patrick Brouwers",
-      "email": "patrick@maatwebsite.nl"
+    "name": "laravel-doctrine/migrations",
+    "type": "library",
+    "description": "Doctrine Migrations for Laravel",
+    "license": "MIT",
+    "keywords": [
+        "doctrine",
+        "laravel",
+        "orm",
+        "data mapper",
+        "database",
+        "migrations"
+    ],
+    "authors": [
+        {
+            "name": "Patrick Brouwers",
+            "email": "patrick@maatwebsite.nl"
+        }
+    ],
+    "repositories": [
+        {
+            "type": "vcs",
+            "url": "https://github.com/77web/laravel-doctrine-orm.git"
```

Comment:
👀

Reply:
to use laravel-doctrine/orm v11 https://github.com/laravel-doctrine/orm/pull/575

Reply:
I see 😄 I have tagged 2.1.0 in laravel-doctrine now, so you should be able to use that 🙂

Reply:
https://github.com/laravel-doctrine/orm/pull/575 is merged right now, so I switched back the dependency to laravel-doctrine/orm [72afa16](https://github.com/laravel-doctrine/migrations/pull/135/commits/72afa16587694b3b3e6195711db6d1a08fd37241)